### ISSUE TITLE
USER ISSUE: It is SO hard to tell what block is blocking you from placing a big object!

### ISSUE THREAD

**Issue by thepolypusher** [StrangeLoopGames/EcoIssues#18953]
\
**Version:** 0.9.0.5 beta release-100

**Steps to Reproduce:**
1. Clear space for something big like a waterwheel
2. Have a friend place a glass window block randomly in the area that you can't quite see
3. Try to place the object
4. Try to figure out where the F that blocking piece is

**Expected behavior:**
The offending blocks should be temporarily highlighted red so that you know what to deal with

**Actual behavior:**
The only feedback you get is 'placing that is blocked dude, good luck figuring out what to do from here.''

**Do you have mods installed? Does the issue happen when no mods are installed?:**

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by ThePiachu** [comment]
Agreed, it would be nice to be able to see say, some outline of what space the building wants to go (I'm looking at you, Assembly Line!), plus be able to look around while trying to place the building (Lumber Stockpiles would benefit from this)...

**Comment by IcedForge** [comment]
Should be looked into after #18606 is done i like the idea to have some form of box line at the edges of the object

**Comment by ThePiachu** [comment]
It might be a bit easier as well if you could build from a bit further away, a bit like Satisfactory. That way you could line up corners a bit easier and so on. Bounding box would still be useful for machines that don't have intuitive sizes, like again that Assembly Line (it looks like you could place it right up against a wall, but nope, you have to have 1 "free" space that's a part of the machine.
